pynicotine/buddies.py

```diff
@@ -165,7 +165,7 @@ def add_buddy(self, username):
         events.emit("add-buddy", username, user_data)
 
         # Request user status, speed and number of shared files
-        core.users.watch_user(username)
+        core.users.watch_user(username, mode="userlist")
```

Comment:
Ambiguity with normal user. "userlist" -> "buddy"

Reply:
Rather than "buddy", actually "buddies" would match what is used in `core.enabled_components`.